### DISCUSSION TITLE
Invalid `prisma.issue.create()` invocation:

### DISCUSSION THREAD

**Discussion by Raunak0713** [prisma/prisma#23435] [Q&A]
### Bug description

Im using prisma with mongodb and when I try to push a new user it says invalid prisma.issue.create(). My github repo is https://github.com/Raunak0713/Issue-Tracker . Im using create method in app/api/issues/route.ts

Thank you

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by Jolg42** [comment]
Hi @Raunak0713, I'm not sure if this is a real bug with the current information you provided.

Please fill in the template and provide the information needed, also make sure to include the full error message.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
